trunk/Bricksmith/Source/Application/General/RelatedParts.m

```diff
@@ -332,6 +332,13 @@ - (id)			initWithFilePath:(NSString *)filePath
 			else
 			{
 #if DEBUG
+			// Where related.ldr includes comments throughout, it ends up
+			// filling the console with output we probably don't care about.
+			// Filtering for the "!" prefix may be overly generic if we want to
+			// actually catch something unexpected here, but as the majority of
+			// meta commands we could possibly care about include the prefix, it
+			// seems like a good balance.
+			if ([parsedField containsString:@"!"])
 				printf("Unparsable META command: %s\n", [orig_line UTF8String]);
```

Comment:
As I’m not sure what the actual intention of this logging is, it may be that something just needs updated so that normal meta commands aren’t reaching this; or maybe this logging should be removed altogether?

I was just having trouble doing some debugging with the console filled with every comment or meta command in the related parts file so I did this so I wouldn’t have to deal with it. If there is something else you would rather do let me know.